process/process.go

```diff
@@ -34,6 +35,21 @@ type systemProcess struct {
 
 var _ Process = &systemProcess{}
 
+var bufPool sync.Pool
+
+// mappingParseBufferSize defines the initial buffer size
+// to parse mappings.
+const mappingParseBufferSize = 512
```

Comment:
On my system, the average line length in `/proc/PID/maps` is 120-130 bytes, minimum 119 and maximum 157, so we should probably reduce this to 256.

```suggestion
const mappingParseBufferSize = 256
```

```
/proc# for pid in [0-9]*; do (cd $pid; wc -L maps | cut -f1 -d ' '); done | sort -n | grep -v 0
119
119
119
119
119
119
119
121
121
123
123
124
124
124
124
124
124
124
124
124
124
125
125
128
128
128
128
128
128
128
128
128
128
128
128
128
128
128
131
131
131
131
131
131
131
131
131
157
```